docs/index.html

```diff
@@ -91,12 +98,11 @@ <h2 class="h4 features__title">Why Us?</h2>
       </div>
       <a href="#" class="button button--lg features__button">Contact Us</a>
     </section>
-    <section class="team">
+    <section class="team" id="staff">
       <h2 class="h4 text-center">Our Team</h2>
       <hr />
       <!-- below section elements are created dynamically with javascript -->
-      <div class="layout-grid" id="team">
-      </div>
+      <div class="layout-grid" id="team"></div>
     </section>
     <section class="partner">
```

Comment:
- [ ] It looks like you don't have the footer section on your page. you should always put a footer on all of your web pages regardless of the design.

You should provide a clear and consistent layout. For best results, you should display a consistent layout with a 'header' as well as a clear and consistent 'footer' for your page. The footer is important as it clearly shows the end of your page and contains legal information about your page. 

You should check the design and implement the footer for both of your pages.